runtime/layers/fpm-dev/Dockerfile

```
@@ -25,10 +29,9 @@ USER root
 COPY --from=build_extensions /tmp/*.so /tmp/
 RUN cp /tmp/*.so $(php -r "echo ini_get('extension_dir');")
 
-# Run PHP-FPM
-# opcache.validate_timestamps=1 : cancels the flag in the base configuration so that files are reloaded
-CMD /opt/bin/php-fpm --nodaemonize --fpm-config /opt/bref/etc/php-fpm.conf -d opcache.validate_timestamps=1 --force-stderr
-
 FROM without_extensions
 
 COPY --from=build_dev  /opt /opt
```

This is a bit sad to copy `php-fpm.conf` twice, I just wonder if we cannot refine this one to:
```suggestion
COPY --from=build_dev  /opt/bref/lib/php/extensions/ /opt/bref/lib/php/extensions/
```

This would allow to remove the COPY instruction below, WDYT?

---

TBH I have no problem with a duplicate line or something that isn't 100% perfect for a temporary solution (when PHP 8.1 and xdebug support are here I guess this can be all simplified).